src/seedu/addressbook/data/person/Address.java

```diff
@@ -14,7 +14,12 @@
 
     public final String value;
     private boolean isPrivate;
-
+    
+    private Block Block;
```

Comment:
As rightly pointed out by your fellow teammates, the first letter shouldn't be capital for variable names.